util/generate-sample-ledger/src/bin/generate_sample_ledger.rs

```diff
@@ -20,11 +21,17 @@ struct Config {
     // Seed to use when generating blocks (e.g. 1234567812345678123456781234567812345678123456781234567812345678).
     #[structopt(long = "seed", short = "s", parse(try_from_str=hex::FromHex::from_hex))]
     pub seed: Option<[u8; 32]>,
+
+    #[structopt(long = "hint-text")]
+    pub hint_text: Option<String>,
 }
 
 fn main() {
     let config = Config::from_args();
 
+    mc_common::setup_panic_handler();
+    let (logger, _global_logger_guard) = create_app_logger(o!());
```

Comment:
recommend to use `create_stderr_logger` for command line tools that are not servers, or you probably won't see the log messages (because the program will close on exit and not flush the buffer)

Reply:
Interesting - I do actually see the log output with the current configuration

Reply:
what we do elsewhere is like this:

```
use mc_common::logger::create_root_logger;

...
fn main() {
    // Logging must go to stderr to not interfere with STDOUT
    std::env::set_var("MC_LOG_STDERR", "1");
...
    mc_watcher::watcher_db::create_or_open_rw_watcher_db(config.watcher, &[], create_root_logger())
}
```

Reply:
you probably see some but not the parts at the end, you would have to put a `thread::sleep` before exiting main

Reply:
Switched to create_root_logger (which wraps create_stderr_logger)